libs/core/src/components/pds-copytext/pds-copytext.tsx

```diff
@@ -43,24 +43,12 @@ export class PdsCopytext {
 
   private copyToClipboard = async (value: string) => {
     try {
-      if (navigator.clipboard) {
+      if (typeof navigator.clipboard !== 'undefined') {
         await navigator.clipboard.writeText(value);
         this.pdsCopyTextClick.emit('Copied to clipboard');
-      } else {
-        // falback for safari
-        const el = document.createElement('textarea');
-        el.value = value;
-        el.setAttribute('readonly', '');
-        el.style.position = 'absolute';
-        el.style.left = '-9999px';
-        document.body.appendChild(el);
-        el.focus();
-        el.setSelectionRange(0, el.value.length);
-        document.execCommand('copy');
-        document.body.removeChild(el);
-        this.pdsCopyTextClick.emit('Copied to clipboard');
       }
     } catch (err) {
+      console.error(err);
```

Comment:
Should this error only be outputted if we are running locally? If so, when can wrap this in a `Build.isDev`

You will need to add Build to the import statement for `@stencil/core`

Reply:
All good! I can just remove this.